src/msw/textctrl.cpp

```diff
@@ -1114,6 +1114,115 @@ void wxTextCtrl::DoSetValue(const wxString& value, int flags)
     }
 }
 
+#if wxUSE_RICHEDIT
+
+DWORD wxCALLBACK MSWEditStreamOutCallback(DWORD_PTR dwCookie, LPBYTE pbBuff,
+    LONG cb, LONG* WXUNUSED(pcb))
```

Comment:
Don't we need to set `*pcb = cb` to indicate that we handled all the bytes?

Reply:
Indeed. I'll fix that.